src/features/movies/MovieList/styled.js

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 324px);
+  grid-gap: 24px;
+  justify-content: center;
+  margin: 12px auto 40px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}){
+    grid-template-columns: 1fr;
+    margin-bottom: 32px;
+  }
```

Comment:
There should be grid-gap: 16px as in the project. You can use this property on small breakpoint.